content/spin/v2/key-value-store-tutorial.md

```diff
@@ -519,6 +541,8 @@ We want to get feedback on the ergonomics of the key value API. We are curious a
 
 You can read the [improvement proposal for key/value support](https://github.com/fermyon/spin/blob/main/docs/content/sips/010-key-value.md) as well as the implementation for the [current feature](https://github.com/fermyon/spin/pull/1035). Please feel free to ask questions and also share your thoughts in [our Discord community](https://discord.gg/AAFNfS7NGf).
 
+
+<!-- TODO Should this be kept around? -->
```

Comment:
@tpmccallum / @karthik2804 - Is this required to stay in this doc?

Reply:
Have removed this from the doc.